server/datastore/mysql/apple_mdm_test.go

```diff
@@ -3326,6 +3326,60 @@ func testMDMAppleSetupAssistant(t *testing.T, ds *Datastore) {
 	require.Equal(t, "test3", noTeamAsst2.Name)
 	require.JSONEq(t, `{"x": 3}`, string(noTeamAsst2.Profile))
 
+	// upsert team no change, uploaded at timestamp does not change
```

Comment:
Should the test wait a second to confirm that the lack of change is not due to the imprecision of the timestamp?

Reply:
Ah yeah, I did add some waits in the worker tests but did not add them here, I'll add those.